src/go/rpk/pkg/config/config.go

```diff
@@ -34,6 +35,46 @@ const (
 	ModeProd = "prod"
 )
 
+func InitViper(fs afero.Fs) *viper.Viper {
+	v := viper.New()
+	v.SetFs(fs)
+	v.SetConfigName("redpanda")
+	v.SetConfigType("yaml")
+	setDefaults(v)
+	return v
+}
+
+func addConfigPaths(v *viper.Viper) {
+	v.AddConfigPath("$HOME")
+	v.AddConfigPath(fp.Join("etc", "redpanda"))
+	v.AddConfigPath(".")
+	path, err := os.Getwd()
+	if err != nil {
+		log.Debugf("Error getting the current dir: %v", err)
```

Comment:
Is this error only debug-level worthy? When can it happen?
OTOH I'd say its at least warn level if not abort :) .

Reply:
Sounds good.